packages/react-magma-dom/src/components/FormFieldContainer/FormFieldContainer.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { axe } from 'jest-axe';
+import { FormFieldContainer } from '.';
+import { render } from '@testing-library/react';
+import { I18nContext, defaultI18n } from '../../i18n';
+
+const TEXT = 'Test Text';
+
```

Comment:
No tests for error or helper messages. And no tests for `inputSize` or any of the `style` overrides.